lib/index.js

```diff
@@ -14,13 +14,23 @@ var mapper = require('./mapper');
 
 var Appboy = module.exports = integration('Appboy')
   .channels(['server', 'client'])
-  .endpoint('https://api.appboy.com/users/track')
+  .ensure('settings.datacenter')
   .ensure('settings.appGroupId')
   .ensure('message.userId')
   .mapper(mapper)
   .retries(2);
 
 /**
+ * Initialize.
+ *
+ * @api private
+ */
+
+Appboy.prototype.initialize = function(){
+  this.endpoint = this.settings.datacenter === 'us' ? 'https://api.appboy.com/users/track' : 'https://rest.api.appboy.eu/users/track';
```

Comment:
Did we migrate existing users? If not (and we're not planning to), maybe the logic should be reversed  (i..e `this.settings.datacenter === 'eu'`) because otherwise not having a setting will use the EU endpoint (instead of the old .com endpoint).